dandischema/models.py

```diff
@@ -49,7 +49,7 @@
     "[a-f0-9]{8}[-]*[a-f0-9]{4}[-]*" "[a-f0-9]{4}[-]*[a-f0-9]{4}[-]*[a-f0-9]{12}$"
 )
 ASSET_UUID_PATTERN = r"^dandiasset:" + UUID_PATTERN
-DANDI_DOI_PATTERN = r"^10.80507/dandi\.\d{6}/\d+\.\d+\.\d+"
+DANDI_DOI_PATTERN = r"^10.[0-9]{5}/dandi\.\d{6}/\d+\.\d+\.\d+"
```

Comment:
@satra -- do we want to limit may be to the only two we have? in the future we should relax it anyways to allow for truly arbitrary DOI prefix I guess

Reply:
i see no reason not too at this point.